src/Apps/Artist/Routes/CV/CVItem.tsx

```diff
@@ -3,7 +3,7 @@ import { groupBy } from "lodash"
 import React, { Component } from "react"
 import styled from "styled-components"
 import { Col, Row } from "Styleguide/Elements/Grid"
-import { Responsive } from "Utils/Responsive"
+import { Media, Responsive } from "Utils/Responsive"
```

Comment:
So I had a go at converting this. Couldn't get it 100%, but the only differences will be attributes.